v1/transformers/terraform/azure/aks.cue

```diff
@@ -79,6 +87,21 @@ import (
 				}
 
 			}
+
+			// Create Subnet For AKS
+			azurerm_subnet: "\(k8s.name)_aks_subnet": {
+			  name                 : "\(k8s.name)-aks-subnet"
+			  resource_group_name  : azure.resourceGroupName
+			  virtual_network_name : azure.vnetName
+			  address_prefixes     : azure.addressAKS
+			}
```

Comment:
Check if inputs exist (vnetName, addressAKS)  and then create if both inputs exist